Foundation/NSURLSession/NSURLSessionTask.swift

```diff
@@ -26,55 +26,19 @@ import Dispatch
 /// of processing a given request.
 open class URLSessionTask : NSObject, NSCopying {
     /// How many times the task has been suspended, 0 indicating a running task.
-    fileprivate var suspendCount = 1
-    fileprivate var easyHandle: _EasyHandle!
-    fileprivate var totalDownloaded = 0
-    fileprivate var session: URLSessionProtocol! //change to nil when task completes
-    fileprivate let body: _Body
-    fileprivate let tempFileURL: URL
+    internal var suspendCount = 1
+    internal var totalDownloaded = 0
```

Comment:
This is specific to the `http download tasks`. We'd need to consider moving it to the HTTP implementation.